From 161bf1cd24b41a8eec86152eee632598fdf50144 Mon Sep 17 00:00:00 2001
From: Phillip Lougher <phillip@squashfs.org.uk>
Date: Sat, 30 Aug 2014 03:04:11 +0100
Subject: [PATCH 081/120] actions: eval_expr_top() should be using subpath and
 not pathname

Subpath is the path rooted at the root of the destination filesystem.
It includes renames (due to name conflicts), and moves.

Pathname is the original source pathname rooted at whatever pathname(s)
were given on the command line.

Signed-off-by: Phillip Lougher <phillip@squashfs.org.uk>
---
 squashfs-tools/action.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/squashfs-tools/action.c b/squashfs-tools/action.c
index 219df2e..5275466 100644
--- a/squashfs-tools/action.c
+++ b/squashfs-tools/action.c
@@ -748,8 +748,8 @@ static int eval_expr_top(struct action *action, struct action_data *action_data)
 
 		expr_log_cmnd(LOG_ENABLE);
 
-		if(action_data->pathname)
-			expr_log(action_data->pathname);
+		if(action_data->subpath)
+			expr_log(action_data->subpath);
 
 		expr_log("=");
 		expr_log(action->action->name);
-- 
2.8.3

